RELEASE.md

```diff
@@ -83,3 +86,56 @@ The usual steps are:
     * Copy the bundle files from `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle`
     * Copy `github.com/kuadrant/kuadrant-operator/tree/v0.W.Z/bundle.Dockerfile` with the proper fix to the COPY commands
       (i.e. remove /bundle from the paths)
+
+--- 
+
+## Release file format.
+
+This example of the `release.toml` file uses tag `v1.0.1` as reference.
+
+```toml
+# FILE: ./release.toml
+[kuadrant]
+channels = ["alpha",]
+release = "1.0.1"
+
+[dependencies]
+Authorino_bundle = "0.16.0"
+Console_plugin = "0.0.14"
+DNS_bundle = "0.12.0"
+Limitador_bundle = "0.12.1"
+Wasm_shim = "0.8.1"
+
+```
+
+```sh
+git checkout v1.0.1
+cat make/release.mk
+
+# FILE: ./make/release.mk
+#Release default values
+AUTHORINO_OPERATOR_VERSION=0.16.0
+AUTHORINO_OPERATOR_BUNDLE_IMG=quay.io/kuadrant/authorino-operator-bundle:v0.16.0
+LIMITADOR_OPERATOR_VERSION=0.12.1
+LIMITADOR_OPERATOR_BUNDLE_IMG=quay.io/kuadrant/limitador-operator-bundle:v0.12.1
+DNS_OPERATOR_VERSION=0.12.0
+DNS_OPERATOR_BUNDLE_IMG=quay.io/kuadrant/dns-operator-bundle:v0.12.0
+WASM_SHIM_VERSION=0.8.1
+RELATED_IMAGE_WASMSHIM=oci://quay.io/kuadrant/wasm-shim:v0.8.1
+RELATED_IMAGE_CONSOLEPLUGIN=quay.io/kuadrant/console-plugin:v0.0.14
+IMG=quay.io/kuadrant/kuadrant-operator:v1.0.1
+BUNDLE_VERSION=1.0.1
+BUNDLE_IMG=quay.io/kuadrant/kuadrant-operator-bundle:v1.0.1
+CATALOG_IMG=quay.io/kuadrant/kuadrant-operator-catalog:v1.0.1
+CHANNELS=alpha
+BUNDLE_CHANNELS=--channels=alpha
+VERSION=1.0.1
+```
+
+## Automated Verification Processes.
+
+* Check that dependencies release images exist for stated version
+* Check that the git tag does not all ready exist for the stated release.
+* Automated test, install via OLM and ensure the correct images are on cluster.
+This should handle most of [Verify OLM Deployment](#verify-olm-deployment), except the checking of the logs.
+* Ensure all required fields are stated in the `release.toml`.
```

Comment:
Tracking checks that I believe should be part of the release process. This section may not make it to the final version.

Reply:
Yeah these look good and IMO should be added as tasks to the overall top level issue

Reply:
Sure, that makes more sense than making the note here.